media/js/privacy/privacy-day.js

```diff
@@ -5,10 +5,88 @@
 $(function() {
     'use strict';
 
-    // Hide the download button from Firefox users
-    if (isFirefox()) {
-        $('#download').hide();
-    } else {
-        $('#principles').hide();
-    }
+    var pager = Mozilla.Pager.pagers[0];
+
+    // scroll to top of pager when switching tabs so all
+    // tab content is visible after switching tabs
+    pager.$container.on('changePage', function() {
+
+        // Get the current document scroll position
+        var scrollPos = $(document).scrollTop();
+
+        // Get the offset of the top of the pager section
+        var pagerTopPos = $('.pager').first().offset().top;
+
+        // If we're scrolled passed the header, jump back up
+        // to the tabs when the tab is switched
+        if (scrollPos > pagerTopPos) {
+            $('html, body').scrollTop($('.pager').first().offset().top);
```

Comment:
Small perf nit: 

You could cache `$('html, body')` as a var to avoid re-querying the DOM each time a tab changes. Same goes for `$('.pager').first()`